_config/dataservices.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
For the configs that do NOT have secrets in them, I recommend we put all environment configuration in the code repository (rather than the config repository, which would only be used for secrets). This would mean that config changes would be rare.  So, the structure of a json config file with environment specific settings might be:

```
{
  "dev": {
    "port": ":8077"
  },
  "stg": {
    "port": "whatever"
  },
  ...
}
```

The config reading code could then be "smart" about choosing the right configuration based upon the environment name.  We could also have an implicit or explicit "global" or "common" area which would apply to all environments.

Thoughts?